wordpress-mobile/WordPress-iOS
# Comments Detail Redesign

Fixes #18999 

The feature branch for the Comment Detail Redesign. 

To test:
1. Log in to a WP Account
2. In My Site, select Menu
3. Comments
4. Select any comment
5. Check if the designs are matching with Figma: JpHNP0U3s67QojtUoQVkSX-fi-105%3A34824
6. Verify the tracking events for Status Selection: Approve, Pending, Spam, Trash, Delete.
7. Check if the info button opens a bottom sheet with user info
8. Tap on the author URL in the bottom sheet and verify if it opens the web page.
9. Go to Reader > Any Post's Comment (by tapping on the comment icon on posts).
10. Check if anything is sticking out. Info button should still be opening a small box with status options.

TODO: RELEASE NOTES need to be updated before merging. 

Note that there is no Feature Flag so we must merge this PR after the release to have 2 weeks of time to test and fix things. Since it is a redesign, feature flag would overly complicated things. We would need to support 2 versions of the same screen with if-checks in the `CommentDetailViewController`.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
